io/post_fv3.F90

```diff
@@ -3265,6 +3286,18 @@ subroutine set_postvars_fv3(wrt_int_state,mpicomp,setvar_atmfile,   &
 !
 !more fields need to be computed
 !
+
+! write lat/lon of the four corner point for rotated lat-lon grid
+      if(mype == 0 .and. maptype == 207)then
+        write(flatlon,1001)ifhr
+        open(112,file=trim(flatlon),form='formatted',status='unknown')
```

Comment:
@WenMeng-NOAA you may want to check if a unit number(112) is available, this can be done in the future though.

Reply:
@junwang-noaa Do you have a sample for inquiry of an unit number? Thanks!

Reply:
I think you could also use newunit:

```
     open (newunit=iunit, trim(flatlon),form='formatted',status='unknown')
     write(iunit,1002).....
     close(iunit)
```

Reply:
@DeniseWorthen Thanks for the solution. I will add it in my next PR.